### ISSUE TITLE
Upgrade ipcluster plugin to use IPython 0.13

### ISSUE THREAD

**Issue by twiecki** [jtriley/StarCluster#167]
There are some issues with the ipython 0.12 parallel support (one thing is that they can't be used with IPython 0.13). Any plans to upgrade to 0.13?

**Comment by jtriley** [comment]
Yes the new 12.04 AMIs will have 0.13.1 installed. The plugin in the latest github repo already supports 0.13+ just need to push out a release. Hoping to do this by the end of the week.

**Comment by twiecki** [comment]
Fantastic -- thanks for this great tool!

**Comment by twiecki** [comment]
I'm using the most recent development snapshot but still getting 0.12 by default. How can I use the 12.04 AMI?

**Comment by jtriley** [comment]
@twiecki Finishing up installing pandas and then will deploy the new 12.04 AMI to all regions along with 0.94 release

**Comment by twiecki** [comment]
IPython 0.13 is available with the new 12.04 AMI's:

```
>>starcluster listpublic                                                                                                                                                                                                                     
StarCluster - (http://star.mit.edu/cluster) (v. 0.9999)                                                                                                                                                                                      
Software Tools for Academics and Researchers (STAR)                                                                                                                                                                                          
Please submit bug reports to starcluster@mit.edu                                                                                                                                                                                             

>>> Listing all public StarCluster images...                                                                                                                                                                                                 

32bit Images:                                                                                                                                                                                                                                
-------------                                                                                                                                                                                                                                
[0] ami-adc149c4 us-east-1 starcluster-base-ubuntu-12.04-x86 (EBS)                                                                                                                                                                           
[1] ami-899d49e0 us-east-1 starcluster-base-ubuntu-11.10-x86 (EBS)                                                                                                                                                                           
[2] ami-8cf913e5 us-east-1 starcluster-base-ubuntu-10.04-x86-rc3                                                                                                                                                                             
[3] ami-d1c42db8 us-east-1 starcluster-base-ubuntu-9.10-x86-rc8                                                                                                                                                                              
[4] ami-8f9e71e6 us-east-1 starcluster-base-ubuntu-9.04-x86                                                                                                                                                                                  

64bit Images:                                                                                                                                                                                                                                
--------------
[0] ami-5b3fb632 us-east-1 starcluster-base-ubuntu-12.04-x86_64 (EBS)
[1] ami-4583572c us-east-1 starcluster-base-ubuntu-11.10-x86_64-hvm (HVM-EBS)
[2] ami-999d49f0 us-east-1 starcluster-base-ubuntu-11.10-x86_64 (EBS)
[3] ami-0af31963 us-east-1 starcluster-base-ubuntu-10.04-x86_64-rc1
[4] ami-2faa7346 us-east-1 starcluster-base-ubuntu-10.04-x86_64-qiime-1.4.0 (EBS)
[5] ami-8852a0e1 us-east-1 starcluster-base-ubuntu-10.04-x86_64-hadoop
[6] ami-a5c42dcc us-east-1 starcluster-base-ubuntu-9.10-x86_64-rc4
[7] ami-a19e71c8 us-east-1 starcluster-base-ubuntu-9.04-x86_64
[8] ami-06a75a6f us-east-1 starcluster-base-centos-5.4-x86_64-ebs-hvm-gpu-hadoop-rc2 (HVM-EBS)
[9] ami-12b6477b us-east-1 starcluster-base-centos-5.4-x86_64-ebs-hvm-gpu-rc2 (HVM-EBS)
```
